packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -13,6 +13,37 @@ export interface CfnIncludeProps {
    * Both JSON and YAML template formats are supported.
    */
   readonly templateFile: string;
+
+  /**
+   * Specifies the template files that define nested stacks that should be included.
+   *
+   * If your template specifies a stack that isn't included here, it won't be created as a NestedStack
+   * resource, and it won't be accessible from getNestedStack().
+   *
+   * If you include a stack here with an ID that isn't in the template,
+   * template creation will fail and an error will be thrown.
+   *
+   * If you include an ID here that exists in the template but isn't in the template,
+   * template creation will fail and an error will be thrown.
+   */
+  readonly nestedStacks?: { [stackName: string]: CfnIncludeProps };
+}
+
+/**
+ * Returned by getNestedStack(). Contains both the NestedStack and
+ * CfnInclude representations of the template.
+ */
+export interface IncludedNestedStack {
+  /**
+   * The NestedStack object that represents the template, which
+   * respresents the scope of the template.
+   */
+  readonly stack: core.NestedStack;
```

Comment:
I wonder - any reason why this is a `NestedStack`? Maybe just `Stack` is better?

Reply:
Its a nestedStack because the stack it represents is nested. I don't think this stack could not be nested.

Reply:
But `NestedStack` extends `Stack` - so it would be valid from a type-standpoint, and I wonder if there's any reason to surface `NestedStack` here.